docs/codeql/reusables/advanced-query-execution.rst

```diff
@@ -4,15 +4,15 @@
    <https://docs.github.com/en/code-security/codeql-cli/using-the-codeql-cli/using-custom-queries-with-the-codeql-cli#including-query-metadata>`__. You can also execute queries using the following
    plumbing-level subcommands:
    
-   - `database run-queries <../manual/database-run-queries>`__, which
+   - `database run-queries <https://docs.github.com/en/code-security/codeql-cli/codeql-cli-manual/database-run-queries>`__, which
      outputs non-interpreted results in an intermediate binary format called
      :ref:`BQRS <bqrs-file>`.
```

Comment:
Is this `ref` correct? I'm not sure how rst resolves this.

Reply:
Oh...I think it links here: https://github.com/github/semmle-code/blob/aeisenberg/refactor-process-config/ql/docs/codeql/codeql-overview/codeql-glossary.rst#L10

Reply:
I _think_ it's correct. It's modelled on the link in line 4. However, if the file isn't being used any more, it sounds as if it might be better to delete it.

Having checked, it was used in a CodeQL CLI article that's now moved to [docs.github.com](https://docs.github.com/en/code-security/codeql-cli/using-the-codeql-cli/using-custom-queries-with-the-codeql-cli#about-custom-queries-and-the-codeql-cli).